DataFormats/Reconstruction/include/ReconstructionDataFormats/MatchInfoTOF.h

```diff
@@ -25,20 +25,21 @@ namespace dataformats
 {
 class MatchInfoTOF
 {
-  using evGIdx = o2::dataformats::EvIndex<int, o2::dataformats::GlobalTrackID>;
+  //  using evGIdx = o2::dataformats::EvIndex<int, o2::dataformats::GlobalTrackID>;
+  using evGIdx = o2::dataformats::GlobalTrackID;
```

Comment:
perhaps better to rename evGIdx to e.g. GID, since there is not reference on event anymore.

Reply:
Done